content/docs/standard-library/core/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Core
+weight: 5
+---
+The `Core`  section describes the core component's.
+
+# Emitter
+The `Emitter` constantly emits a message that you bind to it using the compiler directive `#bind`.This should be avoided if possible (use const's and literals instead).
```

Comment:
Could you please add that Emitter must always be used in conjunction with `Blocker` or deferred connections. Because we need a way to block it's message until some condition is met